df.history.xml

```diff
@@ -2985,7 +2985,7 @@
         <int32_t init-value='-1'/>
         <int32_t init-value='-1'/>
         <stl-vector type-name='int32_t' comment="Observed size of 4 doesn't allow for pointer"/>
-        <stl-vector/>
+        <stl-vector type-name='int32_t' comment="Observed size of 4 doesn't allow for pointer"/>
```

Comment:
I don't think these comments are strictly necessary - they're informative, but usually an explicit `type-name` is informative enough in this case. (A vector of an unidentified type can just be `<stl-vector/>`)

Reply:
The reason I added the comments is to ensure it's not changed back to pointers just because we have no idea of what the vectors contain (in particular if someone is on the 32 bit version, where pointers are the same size). However, if you think that won't happen I'll remove the comments.